res/values-sq/strings.xml

```diff
@@ -372,65 +372,65 @@
     <string name="_8v8">"8v8"</string>
     <string name="Top_Ranks">"Gradat Top"</string>
     <string name="My_Ranks">"Gradat e mia"</string>
-    <string name="Player_XP">"Lojtar XP"</string>
+    <string name="Player_XP">"Lojtarë XP"</string>
 
-    <string name="W_D_L">"WDL"</string>
-    <string name="MATCH_START">"MATCH FILLIMI"</string>
-    <string name="Clan_Warrior">"Clan luftëtar"</string>
-    <string name="Win_a_Clan_War">"Win një Luftës Clan"</string>
-    <string name="Global">"global"</string>
-    <string name="Spectating">"Spectating"</string>
+    <string name="W_D_L">"W D L"</string>
+    <string name="MATCH_START">"FILLIMI LOJËS"</string>
+    <string name="Clan_Warrior">"Clan Luftëtar"</string>
+    <string name="Win_a_Clan_War">"Fito një Luftë të Klanit"</string>
+    <string name="Global">"Global"</string>
+    <string name="Spectating">"Duke vëzhguar"</string>
 
 
-    <string name="snowflakes_">"snowflakes."</string>
-    <string name="presents_">"paraqet."</string>
-    <string name="Snowflake_Collector">"Snowflake Collector"</string>
-    <string name="Present_Collector">"Collector pranishëm"</string>
+    <string name="snowflakes_">"bora "</string>
+    <string name="presents_">"paraqet "</string>
+    <string name="Snowflake_Collector">"Koleksionist i borës "</string>
+    <string name="Present_Collector">"Koleksionist pranishëm "</string>
```

Comment:
Lines 385, 386 and 388:

The ending of them needs to include a period, not a spacebar.